bin/init.bash

```diff
@@ -65,6 +65,19 @@ replace_set_me(){
 
 }
 
+add_nfs_server_ip()
+{
+  file=$1
+  ip=$2
+
+  if [  "$CK8S_CLOUD_PROVIDER" == "exoscale"  ]
+  then
+    NFS_SERVER_IP=$( jq -r $ip < "${config[infrastructure_file]}")
```

Comment:
Would it make sense to use a more hardcoded path to the infrastructure file here, so we can get rid of `config[infrastructure_file]` from the variables altogether?

Reply:
My idea is to completely remove infra.json. And when needed by nfs you manually add the ips in to the config. Doesnt that make sense? Is infra.json used anywere else?

Reply:
Sure, it makes sense! I just didn't understand that the purpose was to completely remove the file when I started reviewing :slightly_smiling_face: